mpmp/src/model/Model.java

```diff
@@ -1,8 +1,65 @@
 package model;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
 /**
- * Not a thing.
+ * A thing.
```

Comment:
This must definitely stay. No one will understand the reference far in the future.